src/App.tsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  key: string | null,
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    key: null,
+  };
+
+  componentDidMount(): void {
+    document.addEventListener('keyup', this.handleClick);
+  }
+
+  componentWillUnmount(): void {
+    document.removeEventListener('keyup', this.handleClick);
+  }
+
+  handleClick = (event: KeyboardEvent) => {
+    this.setState({ key: event.key });
+  };
+
+  render() {
+    const condition = this.state.key;
```

Comment:
Why do you call this condition?
```suggestion
    const { key } = this.state;
```